hs_core/models.py

```diff
@@ -228,8 +228,8 @@ def get_access_object(user, user_type, user_access):
             "email": user.email,
             "organization": user.userprofile.organization,
             "title": user.userprofile.title,
-            "contributions": len(user.uaccess.owned_resources) if user.is_active else None,
-            "viewable_contributions": user.viewable_contributions if user.is_active else None,
+            "contributions": None,
```

Comment:
`contributions` will always be `None`, even when `include_contributions=True`
It looks like this doesn't impact any functionality at this time, so maybe we don't care. I'm just wondering if it could get confusing down the road. I think when I added `viewable_contributions` I should've made it more clear how those are different from `contributions`...

Reply:
@devincowan
It seems `contributions` is not used in UI anywhere. I was going through my slack conversations with @Maurier couple of months on this, and he had confirmed that this attribute is not used. I should have removed it. Let me remove it.

Reply:
I also realized the flag parameter I added to this function is not needed. Initially, I thought it would prevent extra queries when we access `user.vieweable_contributions`. Since, `user.vieweable_contributions` gets computed outside this function, the flag `include_contributions` of no use. Removed it.